src/TestUtils/src/TestUtils.Appium.UITests/AppiumUITestApp.cs

```diff
@@ -666,7 +667,8 @@ public void Tap(Func<AppQuery, AppWebQuery> query)
 
 		public void TapCoordinates(float x, float y)
 		{
-			throw new NotImplementedException();
+			var ta = new TouchAction(_driver);
+			ta.Tap(x, y).Perform();
 		}
```

Comment:
I implemented this and it appears to be screen coordinates. I am not sure the coordinates of Xamarin UI Test...